src/Goerli-DssSpell.t.sol

```diff
@@ -484,51 +539,332 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+        // may
+        rwaconduitout_007.mate(address(this));
+        rwaconduitout_007.hope(address(this));
+
+        rwaconduitout_007.kiss(address(this));
+        assertEq(rwaconduitout_007.bud(address(this)), 1);
+        rwaconduitout_007.pick(address(this));
+
+        uint256 pushAmount = 100 * WAD; // We push only 100 DAI on Görli
+        rwaconduitout_007.push(pushAmount);
+        rwaconduitout_007.quit();
+
+        assertEq(dai.balanceOf(address(rwaconduitout_007)), 0, "RWA007: Output conduit still holds Dai after push()");
+        assertEq(psmGem.balanceOf(address(this)), pushAmount / daiPsmGemDiffDecimals, "RWA007: Psm GEM not sent to destination after push()");
+        assertEq(dai.balanceOf(address(rwaurn_007)), drawAmount - pushAmount, "RWA007: Dai not sent to destination after push()");
+
+        // END
+        giveAuth(address(end), address(this));
+        end.cage();
+        end.cage("RWA007-A");
+
+        end.skim("RWA007-A", address(rwaurn_007));
+
+        (ink, art) = vat.urns("RWA007-A", address(rwaurn_007));
+        uint256 skimmedInk = drawAmount / 250_000_000;
+        assertEq(ink, 1 * WAD - skimmedInk, "RWA007: wrong ink in urn after skim");
+        assertEq(art, 0, "RWA007: wrong art in urn after skim");
+
+        hevm.warp(block.timestamp + end.wait());
+
+        // Removing the surplus to allow continuing the execution.
+        hevm.store(
+            address(vat),
+            keccak256(abi.encode(address(vow), uint256(5))),
+            bytes32(uint256(0))
+        );
+
+        end.thaw();
+
+        end.flow("RWA007-A");
+
+        giveTokens(address(dai), 1_000_000 * WAD);
+        dai.approve(address(daiJoin), 1_000_000 * WAD);
+        daiJoin.join(address(this), 1_000_000 * WAD);
+
+        vat.hope(address(end));
+        end.pack(1_000_000 * WAD);
+
+        // Check DAI redemption after "cage()"
+        assertEq(vat.gem("RWA007-A", address(this)), 0, "RWA007: wrong vat gem");
+        assertEq(rwagem_007.balanceOf(address(this)), 0, "RWA007: wrong gem balance");
+        end.cash("RWA007-A", 1_000_000 * WAD);
+        assertGt(vat.gem("RWA007-A", address(this)), 0, "RWA007: wrong vat gem after cash");
+        assertEq(rwagem_007.balanceOf(address(this)), 0, "RWA007: wrong gem balance after cash");
+        rwajoin_007.exit(address(this), vat.gem("RWA007-A", address(this)));
+        assertEq(vat.gem("RWA007-A", address(this)), 0, "RWA007: wrong vat gem after exit");
+        assertGt(rwagem_007.balanceOf(address(this)), 0, "RWA007: wrong gem balance after exit");
+    }
+
+    function testRWA007_SPELL_LOCK() public {
+        (uint256 pink, uint256 part) = vat.urns("RWA007-A", address(rwaurn_007));
+        uint256 prevBalance = rwagem_007.balanceOf(address(rwaurn_007.gemJoin()));
+
+        assertEq(part, 0, "RWA007/bad-art-before-spell");
+        assertEq(pink, 0, "RWA007/bad-ink-before-spell");
 
-     function testUpdateTeleportFeeds() public {
```

Comment:
this could be disabled and kept for future use

Reply:
I don't think we need to leave everything in. It's always possible to go back and pull archived stuff back in. This one seems less likely to be used frequently.